c/macOS/src/main.c

```diff
@@ -0,0 +1,90 @@
+//
+//  main.c
+//  mptcp-network-client
+//
+//  Created by Anthony Doeraene on 08/08/2024.
+//
+
+#include <stdio.h>
+#include <string.h>
+#include <Network/Network.h>
+
+dispatch_semaphore_t semaphore;
+
+void get(char *path, nw_connection_t conn, dispatch_queue_t queue){
```

Comment:
(detail: maybe add the `static` keyword, not to have complains from the compiler)

BTW: maybe it is clear for macOS users, but how do you compile this file? It might be good to mention that in the `README` file if that's OK. Also, does it need to be in a separated `src` directory?

Reply:
Not necessarily needed, I'll remove this directory. For the compilation, I'll add a Makefile

Reply:
(I was talking about having `static void get()`, but that's OK, a detail, not an issue for this example!)

Reply:
oops, my bad. Didn't get those warning on macOS so I assumed it would be on the variable ^^